source/dont_be_a_robot_build_the_bot.rst

```diff
@@ -0,0 +1,167 @@
+Don't Be a Robot; Build the Bot
+-------------------------------
+
+I've given this talk as keynote at DjangoCon US 2018. Even though I didn't need to
+submit anything there, I still prepared the abstract and outline. (I don't know
+how I would start preparing the talk without any outline!)
+
+I submitted the proposal to PyCon US 2019, and it was accepted.
```

Comment:
o.k.  as if the proto type was only a thought how could the blank space be used for research notes if now you need two more approvals to start even the interface for memory logs. Then a base platform of any kind to conf. a mother board to run diagnostics on your proto type mother board to figure what company has the chips for completion. now its outta our hands because we just added companies to an idea that hasnt been patented and cant request copyrights if we are not sure the build is cgonna do anything...